index.js

```diff
@@ -80,34 +119,106 @@ module.exports = {
         '@rollup/plugin-babel',
         'decorator-transforms',
 
+        // Dependencies out of date from upstream
+        // (or waiting for the release train to catch up)
         'eslint@latest',
         'eslint-plugin-ember@latest',
         'eslint-plugin-n@latest',
         '@babel/eslint-parser@latest',
+        'ember-template-lint@latest',
```

Comment:
can we remove any install of `@latest` from this PR. If you're aiming to have "at least" a specific version make that a `^` install with a known good version. Especially when things like linters have caused issues in the past

Reply:
done